locale/tr.po

```diff
@@ -631,9 +631,9 @@ msgid "[f] skip 'check()' function of PKGBUILD"
 msgstr "[f] PKGBUILD'deki 'check()' fonksiyonunu atla"
 
 #: pikaur/install_cli.py:284
-#, fuzzy, python-brace-format
+#, python-brace-format
 msgid "[f] skip {quote}check(){quote} function of PKGBUILD"
-msgstr "[f] PKGBUILD'deki 'check()' fonksiyonunu atla"
+msgstr "[f] PKGBUILD'deki {quote}check(){quote} fonksiyonunu atla"
```

Comment:
this was an unintentional change, i fixed it in 1dc68f48c7345de747700e8658ce81c353782e3c